frontend/src/js/components/header/devicecount.tsx

```diff
@@ -0,0 +1,66 @@
+import { DeveloperBoard as DeveloperBoardIcon, Warning as WarningIcon } from '@mui/icons-material';
```

Comment:
This I missed earlier: there needs to be a license header at the top... I'll add a task to check this in CI, since I missed this myself at times 🙃